opensrp-app/src/main/java/org/smartregister/clientandeventmodel/Event.java

```diff
@@ -110,7 +110,10 @@ public Event(String baseEntityId, HashMap<String, String> identifiers, String ev
      * @return
      */
     public void setObs(List<Obs> obs) {
-        this.obs = obs;
+        if (obs == null)
+            this.obs = null;
```

Comment:
This null check is not necessary as the obs field is always null. You can just check if  `obs` is not null.